docs/shp.md

```diff
@@ -0,0 +1,35 @@
+## shp
+
+Command-line client for Shipwright's Build API.
+
+```
+shp [command] [resource] [flags]
+```
+
+### Options
+
+```
+      --as string                      Username to impersonate for the operation
+      --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
+      --cache-dir string               Default cache directory (default "/home/vkochuku/.kube/cache")
```

Comment:
During the processing of the Cobra commands, the `cache-dir` is being extended and therefore your home folder path is shown as part of the documentation. Please, check how we could have something like `~/.kube/cache` instead.